src/test/java/hw2/base/BaseTest.java

```diff
@@ -0,0 +1,155 @@
+        }
+    }
+
+    public void assertFrameButton() {
+        WebElement frame = driver.findElement(By.id("frame"));
+        boolean existedFrame = frame.isEnabled();
+        Assert.assertTrue(existedFrame);
+    }
+
+    public void switchToTheFrameAndCheck() {
+        WebElement frame = driver.findElement(By.id("frame"));
+        driver.switchTo().frame(frame);
+        WebElement iFrameButton = driver.findElement(By.id("button-frame"));
+        assertTrue(iFrameButton.isDisplayed());
+    }
+
+    public void switchToOriginalWindowBack() {
+        driver.switchTo().defaultContent();
+    }
+
+    public void assertItemsOnTheLeftSection(WebElement assertionData, String expectedHeaders) {
+        assertTrue(assertionData.isDisplayed());
+        assertEquals(assertionData.getText(), expectedHeaders);
+    }
+
+    //ex2 steps
+    public void openDifferentElementsPage() {
+        WebElement element = driver.findElement((By.xpath("//a/span[contains(text(),'Service')]")));
+        element.click();
+        driver.findElement((By.xpath("//a[contains(text(), 'Different elements')]")));
+        element.click();
+        driver.get("https://jdi-testing.github.io/jdi-light/different-elements.html");
+    }
+
+    public void selectWaterCheckbox(String input) {
+        driver.findElement(By.xpath(String.format("//*[@class = 'checkbox-row']/label[contains(., '%s')]", input))).click();
+    }
+
+    public void selectWindCheckbox(String input) {
+        driver.findElement(By.xpath(String.format("//*[@class = 'checkbox-row']/label[contains(., '%s')]", input))).click();
+    }
+
+    public void selectRadio(String input) {
+        driver.findElement(By.xpath(String.format("//*[@class = 'checkbox-row']/label[contains(., '%s')]", input))).click();
+    }
+
+    public void selectDropdown(String input) {
+        driver.findElement((By.xpath("//select[@class = 'uui-form-element']"))).click();
+        driver.findElement((By.xpath(String.format("//*[@class = 'uui-form-element']/option[contains(., '%s')]", input)))).click();
+    }
+
+    public void assertCheckboxes(String checkbox, boolean status) {
+        Assert.assertTrue(status);
+        List<WebElement> list = driver.findElements(By.cssSelector("ul.panel-body-list.logs li"));
+        status = list
+                .stream()
+                .map(WebElement::getText)
+                .anyMatch(s -> s.contains(checkbox));
+
+        status = list
+                .stream()
+                .map(WebElement::getText)
+                .allMatch(s -> s.contains(checkbox));
+
+    }
+
+    public void assertRadio(String radio, String typeValue) {
+        List<WebElement> list = driver.findElements(By.cssSelector("ul.panel-body-list.logs li"));
+        typeValue = String.valueOf(list
+                .stream()
+                .map(WebElement::getText)
+                .anyMatch(s -> s.contains(radio)));
+    }
+
+    public void assertInDropdown(String dropdown, String typeValue) {
```

Comment:
I think it's not exactly what we want to check:
you need to check that there are a row in log that you change value in dropdown:
![image](https://user-images.githubusercontent.com/44112289/102103149-afe6d580-3e3d-11eb-9112-914405b195b0.png)